### DISCUSSION TITLE
Join GNOME Circle

### DISCUSSION THREAD

**Discussion by daudix** [Tubefeeder/Pipeline#71] [Ideas]
Submit Tubefeeder to GNOME Circle, there is a few advantages:

- More users
- More contributors

**Comment by Schmiddiii** [comment]
I do not know if Tubefeeder is good enough for GNOME Circle, as:

- Is is popular enough?
- Is it good enough (currently 2.6 stars with five reviews in the GNOME software center. I personally do not really care about the reviews in the software center and think the feedback from the issues/matrix/... is more valuable, but maybe the reviewer might care)
- Does not really follow GNOME HIG (I have not yet read through it and I even created this application before knowing about it). This may be  a part where beeing in GNOME Circle might also help.
- No easy settings to change video player, import subscriptions, ... (may change in the future if I deem this worthy enough).
- (Maybe the most problematic part) MPV by default, they would probably like some GNOME technology by default. To use GNOME technology by default, one of the following will be needed:
    - Add this player inside of the Flatpak (which will make the Flatpak very big).
    - Call players outside the Flatpak by default, but this would probably need a permission that would mark the application as "dangerous" in most app stores.
    - Play the videos inside the application (I really do not want and will not do that as there are so many good video players).

I am not sure if  I would pass the process of beeing accepted as a GNOME Circle application, but it might be worth a try.

**Reply by Schmiddiii** [reply]
I looked through the apps that are currently in the process of reviewing for GNOME Circle and saw that it takes quite some time (months) to go through the full review process and fixing things the reviewer does not like. I do have some other things to do in the next few months so I will likely try applying for GNOME Circle if I have some more freetime.

**Reply by daudix** [reply]
Good luck!

**Reply by ghost** [reply]
> I looked through the apps that are currently in the process of reviewing for GNOME Circle and saw that it takes quite some time (months) to go through the full review process and fixing things the reviewer does not like. I do have some other things to do in the next few months so I will likely try applying for GNOME Circle if I have some more freetime.

Any updates?
